Source/WebCore/bindings/js/JSDOMGlobalObject.h

```diff
@@ -79,7 +79,9 @@ class WEBCORE_EXPORT JSDOMGlobalObject : public JSC::JSGlobalObject {
 
     ScriptExecutionContext* scriptExecutionContext() const;
 
-    static bool canCompileStrings(JSC::JSGlobalObject*, JSC::CompilationType, String, JSC::JSValue);
+    static String codeForEval(JSC::JSGlobalObject*, JSC::JSValue);
+    static bool canCompileStrings(JSC::JSGlobalObject*, JSC::CompilationType, String, const JSC::ArgList&);
```

Comment:
I don't think it's common practice to add links or comments in general to headers in webkit.

These functions are also not necessarily a 1-1 matchup to the specs because of perf optimisations on the JSC side so might be more confusing than helpful?